### ISSUE TITLE
Network query builder performance estimator

### ISSUE THREAD

**Issue by nightsh** [tenders-exposed/elvis-ember#69]
Component present on the QB screen, showing:

* contracts count
  (relevant because it directly influences the total number of nodes / edges)
* CPVs selected
  (it often influences how connected things are in a network, which impacts performance)
* some sort of informed guesstimate on how well a network would perform
  (take into consideration browser, version and whatever we might sniff around)

**Comment by nightsh** [comment]
Linked to tenders-exposed/elvis-backend#32

**Comment by nightsh** [comment]
Blocked by #221

**Comment by victor-nitu** [comment]
This is not blocked anymore. However, we need to think about whether or not we want the backend to compute something for this, or we should just produce the data on the client.

Pinging @georgiana-b, let's discuss how it'd be easier to implement this.

**Comment by chutko** [comment]
the attributes of the client play an important part of performance. i think from backend there should come some evaluation of the queried network's complexity. 

that could be our first step - take variables like number of nodes and connections between them and present user with the network complexity based on these values.

after we have this ready, next step could be considering client's capabilities and calculating approx time for rendering the network

**Comment by georgiana-b** [comment]
Let's come back to this in November

**Comment by victor-nitu** [comment]
Deprecated, won't fix.
